security/appsec/osi-intro/identifying-the-network-layer.md

```diff
@@ -1,66 +1,66 @@
 ---
 author: milesflo
+
 type: normal
+
 category: must-know
-inAlgoPool: false
+
+
 links:
   - >-
     [SANS
     Institute](https://www.sans.org/reading-room/whitepapers/protocols/applying-osi-layer-network-model-information-security-1309){website}
+
 parent: securing-the-data-link-layer
+
 ---
 
 # Identifying the Network Layer
 
-
 ---
 
 ## Content
 
-The Network Layer (Layer 3) is the physical infrastructure that allows the transmission of data over the Internet. These include physical routers, routing tables which use IP addresses for both local and external communication, and next-hops that facilitate the large scale transmission of packets.
-
-
----
-
-## Practice
+The **Network Layer** (Layer 3) is responsible for packet forwarding, including routing through different routers.
 
-What role does the Network Layer play in the OSI 7 Layer Model?
-???
+For example, your computer in London is trying to connect to a server in San Francisco. Because there is more than one path that it can take, the **Network Layer** helps it choose the most efficient one.
 
-- Routing and transmission over the physical topography of the Internet
-- Providing computational power via electronic circuits
-- Transmitting data between two directly connected mediums
-- Packaging data packets into data streams and vice versa for transport
-- Providing a user interface
+> The **Network Layer** relays requests from the **Transport Layer** (Layer 4) to the **Data Link Layer** (Layer 2).
 
+Some functions of the **Network Layer** include:
+- connectionless communication
+  - data packets can travel from sender to recipient, without the recipient having to acknowledge receipt
+- host addressing
+  - each host in the network must have a unique address, commonly known as IP (Internet Protocol) address
+- message forwarding
+  - due to the nature of how networks are partitioned, gateways (or routers) must be used to forward packets between them 
```

Comment:
@mihaiberq does this look okay or should we switch to headings instead of nested lists?

![image](https://user-images.githubusercontent.com/19802557/86469643-b5d67780-bd31-11ea-9b6e-1cf8e194a0c3.png)

Reply:
maybe bold the top-level items?